subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/webserver/webserver.go

```diff
@@ -79,6 +79,9 @@ func NewWebServer(sConfig *si.Config,
 	ws.route(si.BindPath, ws.serve(ws.serveBindPath))
 	ws.route(si.PreemptPath, ws.serve(ws.servePreemptPath))
 	ws.route(si.AffinityGroupsPath, ws.serve(ws.serveAffinityGroups))
+	ws.route(si.ClusterStatusPath, ws.serve(ws.serveClusterStatus))
+	ws.route(si.PhysicalClusterPath, ws.serve(ws.servePCStatus))
```

Comment:
>servePCStatus [](start = 46, length = 13)

Also rename it?
servePCStatus
GetPCStatusHandler
etc

to be PhysicalClusterXXX?

Reply:
Also rename it?
serveVCsStatus
GetVCStatus
GetAllVCsStatusHandler
etc

to be VirtualClusterXXX?




---
In reply to: [392781868](https://github.com/microsoft/pai/pull/4153#discussion_r392781868) [](ancestors = 392781868)